lib/tasks/correct_min_age_values.rake

```diff
@@ -0,0 +1,10 @@
+desc "Alter age values from 0 to 1"
```

Comment:
I'm assuming this is because there are too many logs with age 0 to get their owners to fix them intentionally? If 0's an invalid answer now then surely the logs that have 0 have that erroneously so the correct value might not be 1?

Reply:
I think we have checked that all of these records are children, and that's why we're happy to update these